src/test/java/org/folio/consortia/controller/TenantControllerTest.java

```diff
@@ -41,10 +42,15 @@ class TenantControllerTest extends BaseTest {
   ConsortiumRepository consortiumRepository;
```

Comment:
in TenantControllerTest, all testest changed order (first success cases and second fail cases) and add more test cases